activities/codebase-stewardship/activities.md

```diff
@@ -6,10 +6,12 @@ type: Resource
 
 We provide codebase stewardship at the codebase level. Stewardship activities take place in 4 domains:
 
-* community
-* product marketing
-* quality
-* support
+* [community](#community)
```

Comment:
This order doesn't match the order on the page.

Reply:
I've updated this section by removing the list and rewording the section.

See commit message https://github.com/publiccodenet/about/pull/649/commits/e6083d0580f7f4b2a8e1045d30ca363156c3a47f